src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1495,5 +1494,13 @@ public String toVariableNameString(EnchantmentOffer eo) {
 				.since("2.7")
 				.requiredPlugins("Paper 1.16+"));
 		}
+		if (Skript.classExists("com.destroystokyo.paper.event.entity.EndermanEscapeEvent")) {
+			Classes.registerClass(new EnumClassInfo<>(EndermanEscapeEvent.Reason.class, "endermanescapereason", "enderman escape reasons")
+				.user("enderman escape reasons?")
+				.name("Enderman Escape Reason")
+				.description("Represents the escape reason of an enderman.")
+				.since("INSERT VERSION")
+				.requiredPlugins("Paper"));
```

Comment:
```suggestion
					.user("enderman escape reasons?")
					.name("Enderman Escape Reason")
					.description("Represents the escape reason of an enderman.")
					.requiredPlugins("Paper")
					.since("INSERT VERSION"));
```